assembl/models/auth.py

```diff
@@ -1768,18 +1775,19 @@ def known_languages(self):
 class LanguagePreferenceCollectionWithDefault(LanguagePreferenceCollection):
     """A LanguagePreferenceCollection with a fallback language."""
 
-    def __init__(self, locale_code):
-        self.default_locale = Locale.get_or_create(locale_code)
+    def __init__(self, locale_code, session=None):
+        self.default_locale = Locale.get_or_create(locale_code, session)
 
     def default_locale_code(self):
         return self.default_locale
 
-    def find_locale(self, locale, db=None):
+    def find_locale(self, locale, **kwargs):
```

Comment:
Was it necessary to change db to kwargs here?

Reply:
Ooops, bad copy/pasta. I had previously changed the interface of having `find_locale`, but it simply caused way too many issues with translation services and post creation, so I reverted my changes. Missed this!